crates/motoko/src/lib/vm.rs

```diff
@@ -1274,8 +1274,7 @@ fn stack_cont(core: &mut Core, v: Value) -> Result<Step, Interruption> {
                 _ => Err(Interruption::TypeMismatch),
             },
             Call1(inst, e2) => {
-                let v = core.deref_value(Rc::new(v))?;
```

Comment:
This is an example of why I object to this logic.  I removed it here, and all the tests still passed.

Only after changing the other line, above, did a test fail.